go.mod

```diff
@@ -20,6 +20,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+replace github.com/vmware-tanzu/cartographer-conventions/webhook => ./webhook
```

Comment:
Move up to line 4'ish to make it more prominent.

The replace directive also needs to be added to each sample